spec/controllers/admin/searchgov_domains_controller_spec.rb

```diff
@@ -19,4 +21,16 @@
       end
     end
   end
+
+  # rubocop:disable RSpec/AnyInstance
+  describe 'stop_indexing' do
+    let(:domain) { searchgov_domains(:agency_gov) }
+
+    it 'calls stop_indexing! on domain' do
+      expect_any_instance_of(SearchgovDomain).to receive(:stop_indexing!)
```

Comment:
I usually try to find alternatives to the `expect_any_instance_of` method because of brittleness. Is there an opportunity here to use a mock or stub for this specific instance? 

Not a block either.

Reply:
Me too but in this particular case I couldn't picture how to build the test without it.